<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Marcos Mezo</author>
 <class>ClienteBase</class>
 <widget class="QWidget" name="ClienteBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>574</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Cliente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../bulmages.qrc">
    <normaloff>:/Images/client.svg</normaloff>:/Images/client.svg</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hbox2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../bulmages.qrc">:/Images/client.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_guardar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Guardar cliente</string>
          </property>
          <property name="whatsThis">
           <string>Guardar cliente</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/edit-save.svg</normaloff>:/Images/edit-save.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_eliminar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Borrar cliente</string>
          </property>
          <property name="whatsThis">
           <string>Borrar cliente</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/edit-delete.svg</normaloff>:/Images/edit-delete.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_informe">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Imprimir informe</string>
          </property>
          <property name="whatsThis">
           <string>Imprimir informe</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/document-print.svg</normaloff>:/Images/document-print.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="sp6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>371</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="hbox1">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="sp3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>331</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="mui_tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Tab1">
      <attribute name="title">
       <string>&amp;Datos generales</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel2_5">
         <property name="text">
          <string>C.I.F./N.I.F./N.I.E.:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_cifcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="mui_cifcliente">
         <property name="maxLength">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="sp2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="textLabel2_10">
         <property name="text">
          <string>Codigo de c&amp;liente:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_codcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="2">
        <widget class="QLineEdit" name="mui_codcliente">
         <property name="maxLength">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textLabel2_2">
         <property name="text">
          <string>&amp;Nombre completo:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_nomcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QLineEdit" name="mui_nomcliente">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="textLabel2_2_2">
         <property name="text">
          <string>&amp;Empresa:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_corpcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="QLineEdit" name="mui_corpcliente">
         <property name="maxLength">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="textLabel2_4">
         <property name="text">
          <string>No&amp;mbre comercial:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_nomaltcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="5">
        <widget class="QLineEdit" name="mui_nomaltcliente">
         <property name="maxLength">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="textLabel2_6">
         <property name="text">
          <string>&amp;Direccion:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_dircliente</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="mui_dircliente">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="textLabel2_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Codi&amp;go postal:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_cpcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QLineEdit" name="mui_cpcliente">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="textLabel2_9">
         <property name="text">
          <string>&amp;Poblacion:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_poblcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="mui_poblcliente">
         <property name="maxLength">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="textLabel2_9_2">
         <property name="text">
          <string>Pro&amp;vincia:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_idprovincia</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="4" colspan="2">
        <widget class="BlComboBox" name="mui_idprovincia"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="textLabel2_8">
         <property name="text">
          <string>&amp;Telefono fijo:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_telcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="mui_telcliente">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="textLabel2_8_4">
         <property name="text">
          <string>Numero de fa&amp;x:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_faxcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="4" colspan="2">
        <widget class="QLineEdit" name="mui_faxcliente">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="textLabel2_11">
         <property name="text">
          <string>Tele&amp;fono del trabajo:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_teltrabcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="mui_teltrabcliente">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="textLabel2_8_5">
         <property name="text">
          <string>Telefono &amp;movil:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_movilcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="4" colspan="2">
        <widget class="QLineEdit" name="mui_movilcliente">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="textLabel2_8_2">
         <property name="text">
          <string>Pagina &amp;web:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_urlcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLineEdit" name="mui_urlcliente">
         <property name="maxLength">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="textLabel2_8_3">
         <property name="text">
          <string>Correo electro&amp;nico:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_mailcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="4" colspan="2">
        <widget class="QLineEdit" name="mui_mailcliente">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="textLabel2_8_6">
         <property name="text">
          <string>&amp;Forma de pago:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_idforma_pago</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="BfBuscarFormaPago" name="mui_idforma_pago"/>
       </item>
       <item row="9" column="4" colspan="2">
        <widget class="QCheckBox" name="mui_recargoeqcliente">
         <property name="text">
          <string>Aplicar Recargo de Equivalencia</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="textLabel2_8_7">
         <property name="text">
          <string>&amp;Regimen fiscal:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_regimenfiscalcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="BfBuscarRegimenFiscal" name="mui_regimenfiscalcliente"/>
       </item>
       <item row="11" column="1">
        <spacer name="sp1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOtros">
      <attribute name="title">
       <string>&amp;Otros datos</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="QFrame" name="m_frameplugin">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="textLabel2_4_2_2">
         <property name="text">
          <string>Comen&amp;tarios:</string>
         </property>
         <property name="buddy">
          <cstring>mui_comentcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="textLabel2_4_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>32766</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text">
          <string>Datos de comercio &amp;electronico:</string>
         </property>
         <property name="buddy">
          <cstring>mui_ecommercedatacliente</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTextEdit" name="mui_comentcliente">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QTextEdit" name="mui_ecommercedatacliente">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="textLabel2_4_3">
         <property name="text">
          <string>Datos &amp;bancarios:</string>
         </property>
         <property name="buddy">
          <cstring>mui_bancocliente</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="BlBankEdit" name="mui_bancocliente">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>BfBuscarRegimenFiscal</class>
   <extends>QComboBox</extends>
   <header>bfbuscarregimenfiscal.h</header>
  </customwidget>
  <customwidget>
   <class>BfBuscarFormaPago</class>
   <extends>QComboBox</extends>
   <header>bfbuscarformapago.h</header>
  </customwidget>
  <customwidget>
   <class>BlBankEdit</class>
   <extends>QLineEdit</extends>
   <header>blbankedit.h</header>
  </customwidget>
  <customwidget>
   <class>BlComboBox</class>
   <extends>QComboBox</extends>
   <header>blcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mui_guardar</tabstop>
  <tabstop>mui_eliminar</tabstop>
  <tabstop>mui_informe</tabstop>
  <tabstop>mui_tab</tabstop>
  <tabstop>mui_cifcliente</tabstop>
  <tabstop>mui_codcliente</tabstop>
  <tabstop>mui_nomcliente</tabstop>
  <tabstop>mui_corpcliente</tabstop>
  <tabstop>mui_nomaltcliente</tabstop>
  <tabstop>mui_dircliente</tabstop>
  <tabstop>mui_cpcliente</tabstop>
  <tabstop>mui_poblcliente</tabstop>
  <tabstop>mui_idprovincia</tabstop>
  <tabstop>mui_telcliente</tabstop>
  <tabstop>mui_faxcliente</tabstop>
  <tabstop>mui_teltrabcliente</tabstop>
  <tabstop>mui_movilcliente</tabstop>
  <tabstop>mui_urlcliente</tabstop>
  <tabstop>mui_mailcliente</tabstop>
  <tabstop>mui_idforma_pago</tabstop>
  <tabstop>mui_recargoeqcliente</tabstop>
  <tabstop>mui_regimenfiscalcliente</tabstop>
  <tabstop>mui_comentcliente</tabstop>
  <tabstop>mui_ecommercedatacliente</tabstop>
  <tabstop>mui_bancocliente</tabstop>
  <tabstop>mui_aceptar</tabstop>
  <tabstop>mui_cancelar</tabstop>
 </tabstops>
 <resources>
  <include location="../../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
